src/lib/Input/Input.svelte

```
@@ -60,6 +61,14 @@
     showCount = false
   }: InputProperties = $props();
 
+  /* `for` on a <label> resolves against an element's id, never its name. The label
+     was emitted with for={name} while the field itself carried only name={name},
+     so the association never completed for any caller. Derive the id from name so
+     every existing caller that already passes label + name becomes correctly
+     labelled with no change on their part; an explicit id wins, for callers who
+     need uniqueness across repeated rows or who set no name. */
+  const effectiveId = $derived(id || (name !== '' ? name : undefined));
```

ESLint

[error] undefined is not allowed. Use null or proper type checking instead.

(no-restricted-syntax)